libs/core/src/lib/select/select.component.html

```diff
@@ -1,31 +1,29 @@
-<fd-popover [(isOpen)]="isOpen"
-            (isOpenChange)="isOpenChangeHandle($event)"
-            [options]="popperOptions"
-            [fillControlMode]="fillControlMode"
-            [appendTo]="appendTo"
-            class="fd-select-popover-custom">
-    <fd-popover-control>
-        <ng-container *ngIf="triggerTemplate">
-            <ng-container *ngTemplateOutlet="triggerTemplate; context: {$implicit: this}"></ng-container>
-        </ng-container>
-        <ng-container *ngIf="!triggerTemplate">
-            <button class="fd-select-button-custom"
-                    fd-button
-                    aria-haspopup="true"
-                    [ngClass]="{'fd-button--compact': compact}"
-                    [attr.aria-expanded]="isOpen"
-                    [disabled]="disabled">
-                <span class="fd-select-text-custom">{{triggerValue}}</span>
-            </button>
-        </ng-container>
-    </fd-popover-control>
-    <fd-popover-body
-        class="fd-select-popover-body-custom"
-        [style.maxHeight]="maxHeight || (calculatedMaxHeight + 'px')">
-        <fd-menu>
-            <ul fd-menu-list>
-                <ng-content></ng-content>
-            </ul>
-        </fd-menu>
-    </fd-popover-body>
-</fd-popover>
+<fd-popover [(isOpen)]="isOpen" (isOpenChange)="isOpenChangeHandle($event)" [options]="popperOptions"
```

Comment:
@Lokanathan-k Could you revert formatting changes so we can only see what was changes